CHANGELOG.md

```diff
@@ -5,10 +5,37 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+<<<<<<< release-0.5.0
```

Comment:
There are some of the merge diff markers still in the CHANGELOG.

Reply:
Thanks for catching. When resolving the conflicts from GitHub, I'm pretty sure that I had removed those. Maybe it wasn't clean. I have removed it now.